### PR TITLE
#3317 Appening Querystrings to url in ContentFinderByRedirectUrl

### PR DESCRIPTION
**Pull request by enkelmedia**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes: https://github.com/umbraco/Umbraco-CMS/issues/3317

### Description
If the build in 301 redirect handler kicks in and redirects from a old url this fix will also append any querystring from the incoming request in the redirect. See the issue for more details: 

https://github.com/umbraco/Umbraco-CMS/issues/3317

### CODE REVIEW

**Comment by emmaburstow** [comment]
Hi @enkelmedia,

Thanks for the PR - one of us will review the changes and let you know if we need anything from you before we merge. 

Emma, [Community Pull Request team](https://umbraco.com/blog/introducing-the-new-pr-team/)

**Comment by nul800sebastiaan** [comment]
Thanks @enkelmedia - works like a charm!

File: src/Umbraco.Web/Routing/ContentFinderByRedirectUrl.cs [hunk 1]
```diff
@@ -52,8 +52,12 @@ public bool TryFindContent(PublishedContentRequest contentRequest)
                 return false;
             }
 
+            // Apending any querystring from the incoming request to the redirect url.
+            url = string.IsNullOrEmpty(contentRequest.Uri.Query) ? url : url + contentRequest.Uri.Query;
```
Comment (ronaldbarendse) [inline src/Umbraco.Web/Routing/ContentFinderByRedirectUrl.cs]: A more readable version would be:
```c#
var query = contentRequest.Uri.Query;
if (!string.IsNullOrEmpty(query))
{
    url = url + query;
}
```

Comment (enkelmedia) [reply src/Umbraco.Web/Routing/ContentFinderByRedirectUrl.cs]: Hmm, might be more readable depending on who you ask. For the code in the PR is just as readable but if someone wants to change it I'm all fine with that =D

**Pull request merged**